src/BaseExporter.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/*
+ * This file is part of the Exporter package.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SebastianBergmann\Exporter;
+
+/**
+ * Abstract base class for visualizing PHP variables.
+ *
+ * @package    Exporter
+ * @author     Sebastian Bergmann <sebastian@phpunit.de>
+ * @copyright  Sebastian Bergmann <sebastian@phpunit.de>
+ * @license    http://www.opensource.org/licenses/BSD-3-Clause  The BSD 3-Clause License
+ * @link       https://github.com/sebastianbergmann/exporter
+ */
+abstract class BaseExporter
+{
+    /**
+     * @var Factory
+     */
+    protected $factory;
+
+    /**
+     * @param Factory $factory
+     */
+    public function __construct(Factory $factory)
+    {
+        $this->factory = $factory;
+    }
+
+    /**
+     * Returns whether the exporter can export a given value.
+     *
+     * @param  mixed   $value The value to export.
+     * @return boolean
+     */
+    public function accepts($value)
```

Comment:
IMO, you should force sub-classes to define explicitly what they accept, to avoid mistakes